.pre-commit-config.yaml

```diff
@@ -3,36 +3,57 @@ default_language_version:
   python: "3.9"
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: "v4.1.0"
     hooks:
       - id: check-yaml
+        exclude: .clang-format
       - id: end-of-file-fixer
       - id: trailing-whitespace
+
   - repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: "22.1.0"
     hooks:
       - id: black
+
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.7.8
+    rev: 3.9.2
     hooks:
       - id: flake8
         types: [file, python]
         # only check for unused imports, as the rest is done by black
         args: [--select=F401]
+
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.812
+    rev: v0.931
     hooks:
       - id: mypy
         args:
+          - --show-error-codes
           - --ignore-missing-imports
```

Comment:
These are already defined in mypy settings now, I think

Reply:
No. For the general check, missing imports is ignored, otherwise you'd have a massive list of missing imports. For the packages (Pyodide-build and whatever the other one was), missing imports come from the configuration defined list.